bot/cogs/cyber.py

```diff
@@ -369,6 +374,18 @@ def suffix_from_number(num):
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
 
+        # Level Function Regex.
+        match = self.game_level_regex.match(message.content)
+        if match and message.channel.id == HQ_ID or MOON_ID or FORENSICS_ID:
```

Comment:
So this is parsed as

```python
if match and (message.channel.id == HQ_ID) or (MOON_ID) or (FORENSICS_ID):
```

Now, this will _always_ be true because you've set the variable `MOON_ID` and `FORENSICS_ID` to have non-empty value.

Instead, consider

```python
if match and message.channel.id in [HQ_ID, MOON_ID, FORENSICS_ID]:
```